handlers/new_session.go

```diff
@@ -49,8 +50,27 @@ func NewSession(rw http.ResponseWriter, req *http.Request) {
 		}
 
 	}
-	duration := config.GetDuration(reqDur)
-	s, err := core.SessionNew(userId, duration, stack, stackName, imageName)
+
+	playground := core.PlaygroundFindByDomain(req.Host)
+	if playground == nil {
+		log.Printf("Playground for domain %s was not found!", req.Host)
+		rw.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	var duration time.Duration
```

Comment:
The session duration `could` be less than the default (for tutorials for example) but shouldn't be greater. This local was here: https://github.com/play-with-docker/play-with-docker/pull/215/files#diff-3baf47c64847a8fb8aaa8cc2e088513bL151

Otherwise someone could just start a session that lasts several days.